admin-dev/themes/new-theme/js/app/pages/stock/components/header/filters/filter-component.vue

```diff
@@ -79,19 +78,44 @@
   import PSTags from '@app/widgets/ps-tags';
   import PSTreeItem from '@app/widgets/ps-tree/ps-tree-item';
   import PSTree from '@app/widgets/ps-tree/ps-tree';
-  import { EventBus } from '@app/utils/event-bus';
-  import _ from 'lodash';
+  import {EventBus} from '@app/utils/event-bus';
 
   export default {
-    props: ['placeholder', 'itemID', 'label', 'list'],
+    props: {
+      placeholder: {
+        type: String,
+        required: false,
+        default: '',
+      },
+      itemID: {
+        type: String,
+        required: true,
+      },
+      label: {
+        type: String,
+        required: true,
+      },
+      list: {
+        type: Array,
+        required: true,
+      },
+    },
     computed: {
       isOverview() {
         return this.$route.name === 'overview';
       },
       hasPlaceholder() {
         return !this.tags.length;
       },
-      items() {
+      PSTreeTranslations() {
+        return {
+          expand: this.trans('tree_expand'),
+          reduce: this.trans('tree_reduce'),
+        };
+      },
+    },
+    methods: {
+      getItems() {
```

Comment:
Because you do assignment here, which is not allowed in a computed property https://vuejs.org/v2/guide/computed.html#Computed-Properties
`this.hasChildren = true;` => you are not computed data, you use a method to transform them.